examples/demo_bgp.pp

```diff
@@ -133,13 +128,66 @@
   }
 
   #---------------------------------------------------------------------------#
-  # Configure A BGP Neighbor
+  # Configure BGP IPv4 Neighbors
+  #---------------------------------------------------------------------------#
+  cisco_bgp_neighbor {'55.77 blue 1.1.1.1':
+    ensure                 => present,
+
+    #Properties
+    description            => 'my description',
+    connected_check        => true,
+    capability_negotiation => true,
+    dynamic_capability     => true,
+    ebgp_multihop          => 2,
+    local_as               => 55.77,
+    log_neighbor_changes   => disable,
+    low_memory_exempt      => false,
+    remote_as              => 12,
+    remove_private_as      => 'all',
+    shutdown               => true,
+    suppress_4_byte_as     => true,
+    timers_keepalive       => 90,
+    timers_holdtime        => 270,
+    update_source          => 'ethernet1/1',
+    transport_passive_only => false,
+  }
+
+  cisco_bgp_neighbor {'55.77 blue 2.2.2.2':
+    ensure                 => present,
+
+    #Properties
+    description            => 'my description',
+    connected_check        => true,
+    capability_negotiation => true,
+    dynamic_capability     => true,
+    ebgp_multihop          => 2,
+    local_as               => 55.77,
```

Comment:
Non-blocker: local-as should be different than the global asn. It also should process _after_ remote-as or else it will throw an error. I think for now just remove local-as from here and create a US for forcing the processing order in the provider.